src/edge_containers_cli/autocomplete.py

```diff
@@ -128,16 +127,13 @@ def all_iocs(ctx: typer.Context) -> List[str]:
             docker = Docker().docker
             format = "{{.Names}}"
             command = f"{docker} ps -a --filter label=is_IOC=true --format {format}"
```

Comment:
Should container labels not be made generic? To filter out non IOC services for example

Reply:
Took a look at this and I'm going to leave it in the local deployment code for now.

Need to review how to support general service deployment in the local case.